### PR TITLE
MusicXML improvements

### PR DESCRIPTION
**Pull request by rettinghaus**
Fixes an issue with doubled IDs and doubled page breaks when importing MusicXML.

### CODE REVIEW

File: src/iomusxml.cpp [hunk 1]
```diff
@@ -729,34 +729,34 @@ bool MusicXmlInput::ReadMusicXml(pugi::xml_node root)
                 const std::string groupAbbr
                     = GetContentOfChild(xpathNode.node(), "group-abbreviation[not(@print-object='no')]");
                 if (!groupName.empty()) {
-                    Label *label = new Label();
+                    m_label = new Label();
```
Comment (lpugin) [inline src/iomusxml.cpp:732]: Are you sure m_label is always NULL? You should assert it is. I don't know the details about what is being achieved here but this look quite like the door to a memory leak.

Comment (rettinghaus) [reply src/iomusxml.cpp:736]: You are right. Basically this should never happen, but better to be sure anyway.

Comment (lpugin) [reply src/iomusxml.cpp:736]: I think member pointers should be treated more systematically. That is:
* Set them to NULL in the constructor
* Assert that there are not NULL before assigning them to a new object
* Set them to NULL once added to the tree

Comment (rettinghaus) [reply src/iomusxml.cpp:732]: done.

**Pull request merged**